tiny_dnn/network.h

```diff
@@ -386,6 +386,16 @@ class network {
     }
 
     /**
+     * request to finish an ongoing training
+     *
+     * It is safe to test the current network performance in @a on_batch_enumerate and
+     * @a on_epoch_enumerate callbacks during training,
```

Comment:
I see a comma at the end of the line instead of a period, please correct it if it's not intended.